### ISSUE TITLE
FEATURE: ROI registration

### ISSUE THREAD

**Issue by eunjoy21** [MouseLand/suite2p#1115]
### Feature you'd like to see:

I would like to track the same cells over time. Is there any function that I can register or save the ROI from one image and use it to another by uploading the previous ROI? 

### Attempted alternative approaches:

I used the classifier function, but it didn’t work it out well.

**Comment by RichieHakim** [comment]
https://github.com/RichieHakim/ROICaT

**Comment by SSSRgo** [comment]
> https://github.com/RichieHakim/ROICaT

would you be confident with your project?

**Comment by RichieHakim** [comment]
> > https://github.com/RichieHakim/ROICaT
> 
> would you be confident with your project?

Yes. Many people use it including myself because it produces accurate results and can be used on huge datasets. A manuscript is coming out in the next couple of months (hopefully). Here is the first paper published using it: https://www.nature.com/articles/s41586-023-06810-1, though several more should be coming out this year.

**Comment by SSSRgo** [comment]
> > > https://github.com/RichieHakim/ROICaT
> > 
> > 
> > would you be confident with your project?
> 
> Yes. Many people use it including myself because it produces accurate results and can be used on huge datasets. A manuscript is coming out in the next couple of months (hopefully). Here is the first paper published using it: https://www.nature.com/articles/s41586-023-06810-1, though several more should be coming out this year.

Appreciate it! I'll dive into it

**Comment by SSSRgo** [comment]
> > > https://github.com/RichieHakim/ROICaT
> > 
> > 
> > would you be confident with your project?
> 
> Yes. Many people use it including myself because it produces accurate results and can be used on huge datasets. A manuscript is coming out in the next couple of months (hopefully). Here is the first paper published using it: https://www.nature.com/articles/s41586-023-06810-1, though several more should be coming out this year.

It is a great job. There are 2 bugs in your online app: 1. I can not down your sample data. 2. After I uploading my customized data, your sample data is still in the set and I can not remove it.
